src/rtnl/link/nlas/bridge.rs

```diff
@@ -0,0 +1,600 @@
+// SPDX-License-Identifier: MIT
+
+use crate::{
+    constants::*,
+    nlas::{DefaultNla, Nla, NlaBuffer, NlasIterator, NLA_F_NESTED},
+    parsers::{parse_ip, parse_mac, parse_u16, parse_u16_be, parse_u32, parse_u64, parse_u8},
+    traits::{Emitable, Parseable},
+    DecodeError,
+};
+
+use anyhow::Context;
+use byteorder::{BigEndian, ByteOrder, NativeEndian};
+use std::net::{IpAddr, Ipv4Addr, Ipv6Addr};
+
+const BRIDGE_QUERIER_IP_ADDRESS: u16 = 1;
+const BRIDGE_QUERIER_IP_PORT: u16 = 2;
+const BRIDGE_QUERIER_IP_OTHER_TIMER: u16 = 3;
+// const BRIDGE_QUERIER_PAD: u16 = 4;
```

Comment:
Why is this hidden?

Reply:
Just in case developer ask why the number 4 is missing.
The padding is not required to expose.